test/object-store/util/test_file.cpp

```diff
@@ -324,8 +324,13 @@ TestSyncManager::~TestSyncManager()
 {
     if (m_should_teardown_test_directory) {
         if (!m_base_file_path.empty() && util::File::exists(m_base_file_path)) {
-            m_app->sync_manager()->reset_for_testing();
-            util::try_remove_dir_recursive(m_base_file_path);
+            try {
```

Comment:
try/catch will get as going, we need to improve how we unlink the file on windows, as per discussion with @fealebenpae

Reply:
Yes, `try/catch` here will silence the error about trying to delete a file that's currently in use. While we do need to actually fix this in tests so that we've relinquished and closed realm files by the time we tear down, I think the `try/catch` is a good temporary workaround to get us to run tests. On Windows we can add an explicit build step on CI that clears the test's working directory of any undeleted files if necessary.